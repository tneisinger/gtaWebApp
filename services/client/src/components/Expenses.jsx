import React from 'react';

const Expenses = () => (
  <div>
    <h1>Expenses</h1>
    <hr/><br/>
    <p>Expenses goes here.</p>
  </div>
);

export default Expenses;
