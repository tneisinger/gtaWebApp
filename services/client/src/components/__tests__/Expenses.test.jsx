import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Expenses from '../Expenses';

test('Expenses renders properly', () => {
  const wrapper = shallow(<Expenses/>);
  const element = wrapper.find('p');
  expect(element.length).toBe(1);
  expect(element.text()).toBe('Expenses goes here.');
});

test('Expenses renders a snapshot properly', () => {
  const tree = renderer.create(<Expenses/>).toJSON();
  expect(tree).toMatchSnapshot();
});
