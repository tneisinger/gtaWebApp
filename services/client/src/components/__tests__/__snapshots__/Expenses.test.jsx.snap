// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expenses renders a snapshot properly 1`] = `
<div>
  <h1>
    Expenses
  </h1>
  <hr />
  <br />
  <p>
    Expenses goes here.
  </p>
</div>
`;
